scylla-server/src/utils/message.utils.ts

```diff
@@ -8,9 +8,10 @@ export type ClientMessage = {
 /**
  * The format of the data sent to the client
  */
-type ClientData = {
+export type ClientData = {
```

Comment:
this type is here, and in socket-utils in the frontend as 'ServerData.' If there's an easy way to avoid duplication that'd be nice, but not a huge deal

Reply:
Its like if we had a shared library the packages are independent of each other so we either make a shared folder or just make the types twice